homeassistant/components/sensor/citybikes.py

```diff
@@ -0,0 +1,186 @@
+"""
+Sensor for the CityBikes data.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.citybikes/
+"""
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_NAME, CONF_LATITUDE, CONF_LONGITUDE,
+    ATTR_ATTRIBUTION, ATTR_LATITUDE, ATTR_LONGITUDE, ATTR_FRIENDLY_NAME,
+    LENGTH_METERS, LENGTH_FEET)
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import (
+    Throttle, slugify, location, distance)
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['python-citybikes==0.1.3']
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(seconds=12)  # Arbitrary
```

Comment:
Let's make this 5 minutes, 12 seconds is way too frequent. We'll hammer their API.